base/interactiveutil.jl

```diff
@@ -257,7 +257,7 @@ and type signature to `io` which defaults to `STDOUT`. The ASTs are annotated in
 as to cause "non-leaf" types to be emphasized (if color is available, displayed in red).
 This serves as a warning of potential type instability. Not all non-leaf types are particularly
 problematic for performance, so the results need to be used judiciously.
-See [Manual](:ref:`man-code-warntype`) for more codeinformation.
+See [Manual](:ref:`man-code-warntype`) for more code information.
```

Comment:
no trouble at all, though you may want to add [ci skip] at the end of the commit message to save CI time, assuming you've run `make check-whitespace` locally